app/src/main/res/values/strings.xml

```diff
@@ -13,7 +13,7 @@
     <string name="search">search</string>
     <string name="default_content_description">no description</string>
     <string name="prompt_input">Enter here</string>
-
+    <string name="faqSearch">Search FAQ</string>
```

Comment:
Add this string in the FAQ section
after this comment: https://github.com/Swati4star/Images-to-PDF/blob/master/app/src/main/res/values/strings.xml#L571

PS: No need to update the string position in other files. Only this change is good